pytorch_alternatives/custom_pytorch_nlp/Headline Classifier Local.ipynb

```diff
@@ -149,24 +149,6 @@
     "print(encoded_y)"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "df[\"CATEGORY\"][1]"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "encoded_y[0]"
-   ]
-  },
```

Comment:
Would prefer to fix this by correcting the code to pull out the relevant entry of `df`, rather than removing it... I think `.iloc[0]` would do it?